README.ko.md

```diff
@@ -131,18 +145,63 @@ export class UserModule {}
 
 ### Step 5: Server를 시작하고 CRUD API를 확인할 수 있습니다.
 
--   `GET /users` - retrieves a list of users with pagination
--   `GET /users/:id` - retrieves a single user by ID
--   `POST /users` - creates single or multiple users
--   `PATCH /users/:id` - updates an existing user by ID
--   `DELETE /users/:id` - deletes an existing user by ID
--   `PUT /users/:id` - upserts (update or create) an existing user by ID
--   `POST /users/search` - retrieves a list of users based on complex search criteria
--   `POST /users/:id/recover` - recovers a soft deleted user by ID
+모듈이 초기화되면 다음의 endpoint가 자동으로 생성됩니다.
+
+-   `GET /users` - pagination을 적용하여 user의 목록을 조회합니다.
+-   `GET /users/:id` - ID를 기반으로 단일한 user를 조회합니다.
+-   `POST /users` - 단일한, 혹은 여러 명의 user를 생성합니다.
+-   `PATCH /users/:id` - ID를 기반으로 기존의 user를 수정합니다.
+-   `DELETE /users/:id` - ID를 기반으로 기존의 user를 삭제합니다.
+-   `PUT /users/:id` - ID를 기반으로 기존의 user를 upsert (수정하거나 생성) 합니다.
+-   `POST /users/search` - 복잡한 검색 조건을 적용하여 user의 목록을 조회합니다.
+-   `POST /users/:id/recover` - ID를 기반으로 soft delete 된 user를 복구합니다.
 
 ---
 
-## [Contributors](https://github.com/type-challenges/type-challenges/graphs/contributors)
+## 설정
+
+Crud decorator는 다음과 같은 옵션을 제공합니다.
+
+### entity
+
+(required) controller가 다룰 TypeORM Entity를 지정합니다.
+
+### routes
+
+(optional) 각각의 route를 설정합니다.
+
+모든 route는 다음의 기본 옵션들을 가질 수 있습니다.
+
+```typescript
+import { NestInterceptor, Type } from '@nestjs/common';
+
+interface RouteBaseOption {
+    decorators?: Array<PropertyDecorator | MethodDecorator>;
+    interceptors?: Array<Type<NestInterceptor>>;
+    swagger?: {
+        hide?: boolean;
+        response?: Type<unknown>;
+    };
+}
+```
+
+각각의 route는 아래와 같은 옵션을 가질 수 있습니다.
+
+(추가 예정)
```

Comment:
I hope it`s completed

Reply:
Good, I would add them in Korean & English README